reflex-algos/pom.xml

```diff
@@ -262,6 +262,41 @@
             </activation>
             <build>
                 <plugins>
+
+                    <!--
+                    This plugin defines runtime flag skipPyTests when user provides -Dexec.skip or -Dexec.skip=pytest
```

Comment:
How `exec.skip` is related to python tests skipping? The property seems to be too generic to be referred to the python tests.
BTW - was there any issue with your previous solution? (Defining `skipPyTests` as property)

Reply:
exec.skip is the flag defined for Maven exec plugin. If set to true all the execution of the plugin will be skipped. In our case it is building all python packages.
So i didn't want to add a new flag, but added a value instead.